File: createJson/src/main/java/com/ruzhan/jsonfile/model/Video.java
package com.ruzhan.jsonfile.model;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class Video {

    public String id;
    public String title;
    public String image;
    public String playCount;
    public int playNumber;
    public String playNumberDesc;
    public String playWebUrl;

    public String videoUrl;

    public Video(String id, String title, String image, String playCount, int playNumber,
                 String playNumberDesc, String playWebUrl) {
        this.id = id;
        this.title = title;
        this.image = image;
        this.playCount = playCount;
        this.playNumber = playNumber;
        this.playNumberDesc = playNumberDesc;
        this.playWebUrl = playWebUrl;
    }

    public Video(String id, String title, String image, String playCount, int playNumber,
                 String playNumberDesc, String playWebUrl, String videoUrl) {
        this.id = id;
        this.title = title;
        this.image = image;
        this.playCount = playCount;
        this.playNumber = playNumber;
        this.playNumberDesc = playNumberDesc;
        this.playWebUrl = playWebUrl;
        this.videoUrl = videoUrl;
    }
}

File: createJson/src/main/java/com/ruzhan/jsonfile/model/Video.kt
package com.ruzhan.jsonfile.model

class Video {

    var id: String
    var title: String
    var image: String
    var playCount: String
    var playNumber: Int = 0
    var playNumberDesc: String
    var playWebUrl: String

    var videoUrl: String = ""

    constructor(id: String, title: String, image: String, playCount: String, playNumber: Int,
                playNumberDesc: String, playWebUrl: String) {
        this.id = id
        this.title = title
        this.image = image
        this.playCount = playCount
        this.playNumber = playNumber
        this.playNumberDesc = playNumberDesc
        this.playWebUrl = playWebUrl
    }

    constructor(id: String, title: String, image: String, playCount: String, playNumber: Int,
                playNumberDesc: String, playWebUrl: String, videoUrl: String) {
        this.id = id
        this.title = title
        this.image = image
        this.playCount = playCount
        this.playNumber = playNumber
        this.playNumberDesc = playNumberDesc
        this.playWebUrl = playWebUrl
        this.videoUrl = videoUrl
    }
}
